### PR TITLE
ft[radii]: Now you can provide a list of radii of the bodies

### PR DESCRIPTION
**Pull request by saraybakker1**
Provide a list of radii of the bodies and the numbers to which body their correspond. Comment added with info about the syntax and a small example.

### CODE REVIEW

File: fabrics/helpers/casadiFunctionWrapper.py [hunk 1]
```diff
@@ -67,6 +67,14 @@ def evaluate(self, **kwargs):
                 for j, radius_obst_dyn_j in enumerate(kwargs[key]):
                     radius_dyn_dictionary[f'radius_obst_dynamic_{j}'] = radius_obst_dyn_j
                 argument_dictionary.update(radius_dyn_dictionary)
+            if key == 'radius_body_panda_links' or key == 'radius_body_panda_link':
+                """ The syntax should be: [A list of radii of the bodie, a list of the body numbers]
+                # Example: radius_body_panda_links=[[0.02, 0.02, 0.2], [3, 4, 9]] """
+                radius_body_dictionary = {}
+                for j, radius_body_j in enumerate(kwargs[key][0]):
+                    link_nr = kwargs[key][1][j]
+                    radius_body_dictionary[f'radius_body_panda_link{link_nr}'] = radius_body_j
```
Comment (maxspahn) [inline fabrics/helpers/casadiFunctionWrapper.py:76]: This is specific to the `panda` and does not generalize to other robots. Is there a way to make in generic?

**Review by maxspahn — CHANGES_REQUESTED** [review]
The fabrics package should be generic to all robots.

File: fabrics/helpers/casadiFunctionWrapper.py (hunk 1 above)
Comment (saraybakker1) [reply fabrics/helpers/casadiFunctionWrapper.py:76]: I am not super sure. This feature is not useful for pointmasses (but then it is just ignored), only for kinematic chains/arms. 
Then the whole input naming into compute_action() should be more logical for the radius of the bodies. That might be quite a big change to do for all examples. In boxer.py it is radius_body_ee_link for example. 
I think for now this feature would still be nice to have with radius_body_panda_links and it is just ignored in all other cases.

Comment (maxspahn) [reply fabrics/helpers/casadiFunctionWrapper.py:76]: The point is, that this implementation would not work for the ur5 arm for example. However, this should be feasible. You could for example access all the arguments with a leading 'radius' to extract the arguments for body links.

**Comment by maxspahn** [comment]
@saraybakker1 Let me know if you agree with the changes.

**Pull request merged**